pillar/heroku/xpro.sls

```diff
@@ -0,0 +1,38 @@
+{% set app_name = 'xpro' %}
+{% set environments = ['ci', 'rc', 'production'] %}
+{% set status_token = salt.random.get_str(24) %}
+
+proxy:
+  proxytype: heroku
+
+heroku:
+  {% for env in environments %}
+  app_name: {{ app_name }}-{{ env }}
+  api_key: __vault__::secret-operations/global/heroku-api-key>data>value
+  config_vars:
+    AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/xpro-{{ env }}>data>access_key
+    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/xpro-{{ env }}>data>secret_key
```

Comment:
Is this intended to be a role for xpro to use multiple services, or just for S3? If it's multiple services we will need to define the role in the vault pillar.